pkg/volume/csi/nodeinfomanager/nodeinfomanager_test.go

```diff
@@ -627,6 +657,17 @@ func test(t *testing.T, addNodeInfo bool, csiNodeInfoEnabled bool, testcases []t
 			continue
 		}
 
+		// We are testing max volume limits
+		if tc.inputVolumeLimit > 0 {
+			volumeLimits := getVolumeLimits(node)
+			attachKey := v1.ResourceName(util.GetCSIAttachLimitKey(tc.driverName))
+			attachLimit := volumeLimits[attachKey]
+			if attachLimit != tc.expectedVolumeLimit {
+				t.Errorf("expected volume limit to be %d got %d", tc.inputVolumeLimit, attachLimit)
```

Comment:
The log message is incorrect - expected volume limit is `tc.expectedVolumeLimit` not `tc.inputVolumeLimit`

Reply:
good catch.